src/App.vue

```diff
@@ -19,6 +26,12 @@ export default defineComponent({
   },
   mounted() {
     this.fetchAllMovies();
+  },
+  errorCaptured(err, instance, info) {
+    this.error = err;
+    this.errorInstance = instance;
+    this.errorInfo = info;
+    return false;
```

Comment:
I don't think it is a good idea to actually return `false` here. Errors here are not expected and doing so will prevent global handler from getting this error - so e.g. error monitoring tools will not work properly.

This may have sense for some minor components that are expected to fail sometimes.

Reply:
Ok, thanks for the tip!